Testscripts/Linux/perf_mariadb.sh

```diff
@@ -0,0 +1,285 @@
+}
+
+function run_mariadb () {
+	threads=$1
+
+	LogMsg "======================================"
+	LogMsg "Running mariadb test with current threads: ${threads}"
+	LogMsg "======================================"
+	start_monitor ${threads}
+
+	sysbench ${oltp_path} --mysql-host=${server} --mysql-user=${user} --mysql-password=lisapassword \
+--mysql-db=sbtest --time=${MAX_TIME} --oltp-test-mode=complex --mysql-table-engine=innodb --oltp-read-only=off \
+--max-requests=1000000 --num-threads=${threads} run > "${LOG_FOLDER}/${threads}.${MARIADB_LOG_NAME}"
+
+	parse_log ${threads}
+}
+
+Run_SSHCommand "${server}" "rm -rf ${LOG_FOLDER}"
+Run_SSHCommand "${server}" "mkdir -p ${LOG_FOLDER}"
+rm -rf ${LOG_FOLDER}
+mkdir -p ${LOG_FOLDER}
+
+# Install mariadb in client and server machine
+LogMsg "Configuring client ${client}..."
+install_mariadb
+if [ $? -ne 0 ]; then
+	LogErr "Mariadb installation failed in ${client}.."
+	SetTestStateAborted
+	exit 0
+fi
+
+LogMsg "Configuring server ${server}..."
+Run_SSHCommand "${server}" ". $UTIL_FILE && install_mariadb"
+if [ $? -ne 0 ]; then
+	LogErr "Mariadb installation failed in ${server}..."
+	SetTestStateAborted
+	exit 0
+fi
+
+# Install sysbench in client machine
+LogMsg "Installing sysbench in client ${client}..."
+install_sysbench
+
+# Create data directory of mysql on server machine
+LogMsg "Creating data directory of mysql in server ${server}..."
+create_data_dir
+if [ $? -ne 0 ]; then
+	LogErr "Creating data directory of mysql failed in server ${server}.."
+	SetTestStateAborted
+	exit 0
+fi
+
+# Config MariaDB
+LogMsg "Configing MarinaDB in server ${server}..."
+config_mariadb
+if [ $? -ne 0 ]; then
+	LogErr "Configing MarinaDB failed in server ${server}.."
+	SetTestStateAborted
+	exit 0
+fi
+
+# Config MariaDB for remote access
+LogMsg "Configing MarinaDB for remote access in server ${server}..."
+config_mariadb_for_remote_access
+if [ $? -ne 0 ]; then
+	LogErr "Configing MarinaDB for remote access failed in server ${server}.."
+	SetTestStateAborted
+	exit 0
+fi
+
+# Prepare for mariadb test
+LogMsg "Prepare for MarinaDB test in client ${client}..."
+oltp_path="${HOMEDIR}/sysbench-${SYSBENCH_VERSION}/tests/include/oltp_legacy/oltp.lua"
+sysbench ${oltp_path} --mysql-host=${server} --mysql-user=${user} --mysql-password=lisapassword \
+--mysql-db=sbtest --oltp-table-size=1000000 prepare >> ${COMMON_LOG_FILE}
+
+echo "Threads,TotalQueries,TransactionsPerSec,Latency95Percentile_ms" > "${MARIADB_RESULT}"
+
+# Run mariadb test
+LogMsg "Running MariaDB performance test using sysbench in client ${client}..."
+for threads in "${THREADS[@]}"
+do
+    run_mariadb ${threads}
+done
+
+sysbench ${oltp_path} --mysql-host=${server} --mysql-user=${user} --mysql-password=lisapassword --mysql-db=sbtest cleanup >> ${COMMON_LOG_FILE}
+
+LogMsg "Kernel Version : $(uname -r)"
+LogMsg "Guest OS : ${distro}"
+
+column -s, -t "${MARIADB_RESULT}" > "${LOG_FOLDER}"/report.log
+cp "${LOG_FOLDER}"/* .
+cat report.log
```

Comment:
If this `cat report.log` is for displaying result, should we add it as LogMsg?

Reply:
We don't need to add it as LogMsg. This report.log is also the log we collect.